jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/bars/AppBar.java

```diff
@@ -76,4 +86,27 @@ public boolean isScrolled() {
     public boolean isShrinkOnScroll() {
         return hasClass("v-app-bar--shrink-on-scroll");
     }
+
+    @JDIAction("Check that '{name}' is short")
+    public boolean isBarShort() {
+        return attr("style").contains(BarHeight.SHORT.height);
+    }
+
+    @JDIAction("Check that '{name}' is usual size")
+    public boolean isBarUsualSize() {
+        return attr("style").contains(BarHeight.USUAL.height);
+    }
+
+    public enum BarHeight {
+
+        USUAL("64px"),
+        THRESHOLD("128px"),
+        SHORT("56px");
+
+        private final String height;
+
+        BarHeight(String height) {
+            this.height = height;
+        }
+    }
```

Comment:
Please, investigate possibilities to check 'inverted-scroll'

Reply:
delegated